python-stdlib/tarfile/tarfile/__init__.py

```diff
@@ -1,6 +1,7 @@
 """Subset of cpython tarfile class methods needed to decode tar files."""
 
 import uctypes
+import os
```

Comment:
This import shouldn't be needed anymore.

Reply:
true!
fixing it now

Reply:
done!
I really appreciate your time, @dpgeorge 🙏🏼